versions/latest-version.txt

```diff
@@ -1 +1 @@
-0.83.3
+0.85.2.1
```

Comment:
The version is going backwards?

Reply:
Looks like it's going forward but it's not valid SemVer.

If you can trim it down to `0.85.2` the tests will pass. Or `0.85.2-1` works too.